develop/index.md

```diff
@@ -14,3 +14,4 @@ uid: DevelopIndex
 - <xref:DSI>
 - [DataMiner module integrations](xref:DataMinerModuleIntegrations)
 - <xref:ThirdPartyIntegrations>
+- [Best practices](xref:Best_practices)
```

Comment:
an entry for the newly added pages will also need to be added to the develop/toc.yml

Reply:
will do afterwards. had a merge conflict before.